flink-runtime/src/test/java/org/apache/flink/runtime/blob/BlobClientSslTest.java

```diff
@@ -58,7 +58,7 @@ public static void startSSLServer() throws IOException {
 		Configuration config = SSLUtilsTest.createInternalSslConfigWithKeyAndTrustStores();
 		config.setString(BlobServerOptions.STORAGE_DIRECTORY, temporarySslFolder.newFolder().getAbsolutePath());
 
-		blobSslServer = new BlobServer(config, new VoidBlobStore());
+		blobSslServer = new TestBlobServer(config, new VoidBlobStore());
```

Comment:
`BlobClientSslTest` inherits `BlobClientTest`, and it will also execute the new test `testSocketTimeout` added in `BlobClientTest`. If `TestBlobServer` is not used here, an error will occur.